src/query/storages/information-schema/src/key_column_usage_table.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::BTreeMap;
+use std::sync::Arc;
+
+use common_catalog::table::Table;
+use common_meta_app::schema::TableIdent;
+use common_meta_app::schema::TableInfo;
+use common_meta_app::schema::TableMeta;
+use common_storages_view::view_table::ViewTable;
+use common_storages_view::view_table::QUERY;
+
+pub struct KeyColumnUsageTable {}
+
+impl KeyColumnUsageTable {
+    pub fn create(table_id: u64) -> Arc<dyn Table> {
```

Comment:
How about adding a sqllogic test to cover this which can make sure we never break your use cases unintendedly?

Reply:
Added test to `01_0003_information_schema` file, or should I add it a separate file for sharding-jdbc?

Reply:
I think that's ok. But if you want to add test better to add in there https://github.com/datafuselabs/databend/tree/main/tests/suites/1_stateful. Add a new fold.